index.js

```diff
@@ -43,6 +43,26 @@
 //.
 //. Sanctuary is designed to work in Node.js and in ES5-compatible browsers.
 //.
+//. ## Installation
+//.
+//. Sanctuary can be installed via npm:
+//.
+//. ```console
+//. $ npm install sanctuary
+//. ```
+//.
+//. Or simply include a script tag in your html document:
+//.
+//. ```html
+//. <script src="https://unpkg.com/sanctuary@latest/dist/sanctuary.min.js"></script>
```

Comment:
For debugging purposes would it be better to suggest the unminified file here?

Reply:
We could, I was thinking the target audience for the bundle is unlikely to be debugging sanctuary source code.  So I was optimizing for load time.  What do you think?

Reply:
> I was optimizing for load time.

That seems reasonable to me. :)